docs/reference/mobile-manipulation.md

```diff
@@ -0,0 +1,269 @@
+
+    agent = MobileRLLearner(env,
+                            checkpoint_after_iter=20_000,
+                            temp_path=logpath,
+                            device='cpu')
+
+    # train on random goal reaching task
+    agent.fit(env, val_env=eval_env)
+
+    # evaluate on door opening in analytical environement
+    evaluate_on_task(env_config, eval_env_config=eval_config, agent=agent, task='door', world_type='sim')
+
+    rospy.signal_shutdown("We are done")
+  ```
+
+* **Evaluate a pretrained model**.
+  Source the catkin workspace and run the launch file as described in the `ROS Setup` section above. Then run
+  ```python
+    import rospy
+    from pathlib import Path
+    from control.mobile_manipulation.mobileRL.evaluation import evaluate_on_task
+    from control.mobile_manipulation.mobileRL.utils import create_env
+    from control.mobile_manipulation.mobile_manipulation_learner import MobileRLLearner
+
+    # need a node to for visualisation
+    rospy.init_node('kinematic_feasibility_py', anonymous=False)
+
+    main_path = Path(__file__).parent
+    logpath = f"{main_path}/logs/demo_run"
+
+    # create envs
+    eval_config = {
+      'env': 'pr2',
+      'penalty_scaling': 0.01,
+      'time_step': 0.02,
+      'seed': 42,
+      'strategy': 'dirvel',
+      'world_type': 'sim',
+      # set this to true to evaluate in gazebo
+      'init_controllers': False,
+      'perform_collision_check': True,
+      'vis_env': True,
+      'transition_noise_base': 0.0,
+      'ik_fail_thresh': 100,
+      'learn_vel_norm': -1,
+      'slow_down_real_exec': 2,
+      'head_start': 0,
+      'node_handle': 'eval_env',
+      'nr_evaluations': 50,
+    }
+  
+    eval_env = create_env(eval_config, task='rndstartrndgoal', node_handle="eval_env", wrap_in_dummy_vec=True, flatten_obs=True)
+
+    agent = MobileRLLearner(eval_env,
+                            checkpoint_after_iter=0,
+                            checkpoint_load_iter=1_000_000,
+                            restore_model_path='pretrained',
+                            temp_path=logpath,
+                            device='cpu')
+
+    # evaluate on door opening in the analytical environment
+    evaluate_on_task(eval_config, eval_env_config=eval_config, agent=agent, task='door', world_type='sim')
+
+    rospy.signal_shutdown("We are done")
+  ```
+* **Execution in different environments**. 
+  The trained agent and environment can also be directly executed in the real world or the gazebo simulator. For this first start the appropriate ros nodes for your robot. Then pass `world_type='world'` for real world execution or `world_type='gazebo'` for gazebo to the `evaluate_on_task()` function.     
+
+
+#### Notes
+
+##### HSR
+The HSR environment relies on packages that are part of the proprietory HSR simulator. If you have an HSR account with Toyota,
+please follow these steps to use the environment. Otherwise ignore this section to use the other environments we provide.
```

Comment:
```suggestion
The HSR environment relies on packages that are part of the proprietory HSR simulator.
If you have an HSR account with Toyota, please follow these steps to use the environment.
Otherwise ignore this section to use the other environments we provide.
```